plugins/woocommerce/tests/legacy/unit-tests/woocommerce-admin/api/reports-taxes.php

```diff
@@ -60,20 +78,26 @@ public function test_get_reports() {
 		$product->set_regular_price( 25 );
 		$product->save();
 
-		$wpdb->insert(
+		$tax_rate_id = $wpdb->insert(
 			$wpdb->prefix . 'woocommerce_tax_rates',
 			array(
 				'tax_rate_id'       => 1,
 				'tax_rate'          => '7',
 				'tax_rate_country'  => 'US',
-				'tax_rate_state'    => 'GA',
+				'tax_rate_state'    => 'NY',
```

Comment:
Any reason for this change? I mean, I'm partial to NY 😄, but there doesn't seem to be a need for this change.

Reply:
There is a reason, as we make use of the `create_order` helper function here that auto sets the billing/shipping address to NY: https://github.com/woocommerce/woocommerce/blob/e05697dfabe8d5ab6019147729d0858238798b8d/plugins/woocommerce/tests/legacy/framework/helpers/class-wc-helper-order.php#L86 
Meaning that this tax rate won't work if we keep it as GA, as you can see by the below screenshot, that it will fail.
![Screenshot 2024-04-18 at 2 36 23 PM](https://github.com/woocommerce/woocommerce/assets/2240960/81e01e5a-8ae0-4901-bda0-2b539ede379c)